docs/src/docs/usage/false-positives.mdx

```diff
@@ -33,90 +34,83 @@ Exclude issue by text using command-line option `-e` or config option `issues.ex
 It's helpful when you decided to ignore all issues of this type.
```

Comment:
The sentence above should be updated, since we don't have `-e` and `issues.exclude`:
```
Exclude issue by text using command-line option `-e` or config option `issues.exclude`.
```